EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,131 @@
+---
+eip: xxxx
+title: Unified Network Configuration
+description: Execution Layer to fetch network parameters from Consensus Layer
+author: Barnabas Busa (@barnabasbusa), Parithosh Jayanthi (@parithosh), Toni Wahrstätter (@nerolation)
+discussions-to:
```

Comment:
Please create a discussions topic in Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5